PWGHF/D2H/Tasks/CMakeLists.txt

```
@@ -114,6 +114,11 @@ o2physics_add_dpl_workflow(task-omegac0-to-omega-pi
                     PUBLIC_LINK_LIBRARIES O2Physics::AnalysisCore
                     COMPONENT_NAME Analysis)
 
+o2physics_add_dpl_workflow(task-xic0-to-xi-pi-kf
+                    SOURCES taskXic0ToXiPiKf.cxx
```

```suggestion
o2physics_add_dpl_workflow(task-xic0-to-xi-pi
                    SOURCES taskXic0ToXiPi.cxx
```
I wouldn’t include “kf” in the title, since anyone who wants to validate the DCA fitter can easily add the processing function to this task in the future